interpreter/cling/include/cling/Interpreter/CompilationOptions.h

```diff
@@ -12,6 +12,22 @@
 
 namespace cling {
 
+  ///\brief Bits that can be set in `CompilationOptions::IgnoreDiagsMask` to
+  /// filter a group of diagnostics. These flags can be bitwise-OR'd together.
+  enum IgnoreDiags {
+    kNone = 0x00,
+
+    ///\brief Prompt input can look weird for the compiler, e.g.
+    /// void __cling_prompt() { sin(0.1); } // warning: unused function call
+    /// This flag suppresses these warnings; it should be set whenever input
+    /// is wrapped.
+    kPromptBasic = 0x01,
+
+    ///\brief Ignore additional (yet useful for prompt input) warnings, e.g.
+    /// `int (a); // warning: redundant parentheses surrounding declarator`.
+    kPromptExtended = 0x02
```

Comment:
I don't understand from the description nor the name which warnings would be suppressed by `kPromptExtended`.